### PR TITLE
Remove action item codes for Assessments v1

### PR DESCRIPTION
**Pull request by iamtheluckyest**
## Related Issues

- Links https://transcend.height.app/T-34487

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: src/actionItems.ts
Comment (iamtheluckyest) [reply src/actionItems.ts:16]: No, but I think I have to remove them here first, and then I'll update the package in the monorepo and remove them from code there.

Comment (anotherminh) [reply src/actionItems.ts:16]: Ok! It may be tricky if it's the same exact enum used in the frontend and the backend, because we'll want to remove the reference from frontend first, deploy that, then backend..

**Review by anotherminh — DISMISSED** [review]

File: src/actionItems.ts
Comment (iamtheluckyest) [reply src/actionItems.ts:16]: Yeah, part of the difficulty is that a lot of the FE is typed to expect the whole enum to be there, so it throws TS errors if you try to do the FE first.

Comment (iamtheluckyest) [reply src/actionItems.ts:16]: That might work... 🤔

Comment (iamtheluckyest) [reply src/actionItems.ts:16]: Does this plan in the description here sound right? https://transcend.height.app/T-34487

Comment (anotherminh) [reply src/actionItems.ts:16]: Yep! Sounds perfect!!

File: src/actionItems.ts
Comment (anotherminh) [inline src/actionItems.ts:5]: nit: Can we rename the old one to say ActionItemCodeWithDeprecatedEntry and that way the "new" updated enum can have the same name as before?

Comment (iamtheluckyest) [reply src/actionItems.ts:5]: But won't that break the way the import works, as if I had just removed the values from `ActionItemCode` right now?

Comment (anotherminh) [reply src/actionItems.ts:5]: @iamtheluckyest when you import the package into the monorepo, you'd update references to the backend to use the enum `ActionItemCodeWithDeprecatedEntry` so it wouldn't break i think?

Comment (iamtheluckyest) [reply src/actionItems.ts]: It still needs approval to merge though.

**Review by anotherminh — DISMISSED** [review]

File: src/actionItems.ts
Comment (iamtheluckyest) [reply src/actionItems.ts]: It says I dismissed your review but I didn't! 😳 
image

Can you approve again, plz? 🥺

**Review by anotherminh — APPROVED** [review]
@iamtheluckyest it dismisses stale review whenever new commits are pushed!

**Comment by iamtheluckyest** [comment]
> @iamtheluckyest it dismisses stale review whenever new commits are pushed!

@anotherminh oh, really? I had no idea. Is that specific to this repo?

**Pull request merged**
